### DISCUSSION TITLE
What is the current status of trading a portfolio of multiple assets?

### DISCUSSION THREAD

**Discussion by eduardo-ridian** [kernc/backtesting.py#1090] [Q&A]
Hello, Im new here.

Im looking at backtesting.py to replace a backtester I currently have. However I normally think about everything from the portfolio level so I would like to trade multiple assets at once.
I see there are some past talks regarding those topics like: [20](https://github.com/kernc/backtesting.py/issues/20) and [104](https://github.com/kernc/backtesting.py/issues/104).

I was hoping to learn about the current status. Is it something that can be done? Is it on the road map?

Thanks. Big fan of the project.

**Comment by BradKML** [comment]
Seconded this but I think portfolio backtester seems to operate on a different level, but I guess treating a strategy as its own column of value can work? https://github.com/dcajasn/Riskfolio-Lib https://github.com/jankrepl/deepdow https://github.com/skfolio/skfolio https://github.com/cvxgrp/cvxportfolio https://github.com/fmilthaler/FinQuant https://github.com/quantopian/pyfolio https://github.com/ranaroussi/quantstats
